tests/str/tests.py

```diff
@@ -33,3 +33,18 @@ def test_international(self):
             # output of __unicode__() -- or __str__() when the
             # python_2_unicode_compatible decorator is used.
             self.assertEqual(str(a), b'Girl wins \xe2\x82\xac12.500 in lottery')
+
+    def test_defaults(self):
+        """
+        The default implementation of __str__ and __repr__ should return
+        instances of str.
+        """
+        obj = Default()
+        # Explicit call to __str__ to make sure str() doesn't coerce the
+        # returned value.
+        self.assertIsInstance(obj.__str__(), str)
+        self.assertEqual(str(obj), str('Default object'))
+        # Explicit call to __repr__ to make sure repr() doesn't coerce the
```

Comment:
I think you could combine these comments.
`# Explicit call to __str__/__repr__ to make sure str()/repr() don't coerce...`